les/server_handler.go

```diff
@@ -151,19 +153,14 @@ func (h *serverHandler) handle(p *clientPeer) error {
 		p.Log().Debug("Client pool already closed")
 		return p2p.DiscRequested
 	}
-	activeCount, _ := h.server.clientPool.Active()
-	clientConnectionGauge.Update(int64(activeCount))
```

Comment:
Why did you remove these updates? Do you think this metric is useless? Btw the metric itself is still there is `metrics.go` so this looks wrong.

Reply:
Because they are already counted in the clientpool internal. https://github.com/ethereum/go-ethereum/blob/master/les/vflux/server/clientpool.go#L154